.github/test-spec.yml

```diff
@@ -218,3 +218,15 @@
       - "subsys/bluetooth/**/*"
       - "!subsys/bluetooth/mesh/**/*"
       - "!subsys/bluetooth/audio/**/*"
+
+"CI-audio-test":
```

Comment:
Make sure to add:
nrfx
sd_card
zephyr/bluetooth/ in general
zephyr/usb/
zephyr/drivers/watchdog
zephyr/task_wdt/
zephyr/drivers/ (spi, gpio, twi, pwm, I2S)
zephyr/shell/
zephyr/fs/
zephyr/storage/
zephyr/logging/